### ISSUE TITLE
<webview> Fullscreen API doesn't work on 4.x

### ISSUE THREAD

**Issue by khoaakt** [electron/electron#16648]
* Electron version: 4.x
* Operating System (Platform and Version): Windows 10, MacOS 10.14.3

**Expected Behavior**
Fullscreen API methods should be working.
https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API

**Actual behavior**
Nothing happens

**To Reproduce**

```sh
$ git clone https://github.com/hokein/electron-sample-apps.git
$ cd electron-sample-apps/webview/browser
$ npm install
$ npm start || electron .
```
* Go to https://youtube.com/.
* Click on random video, then click fullscreen button.

**Comment by littlecold233** [comment]
Yes, I have the same Problem

I could produce the issue in 4.x version. **I electron 3.x version, the video fullscreen API in webview works well** 

It must be some update in 4.x cause  the issue

**Comment by codebytere** [comment]
@khoaakt there is no webview directory in https://github.com/electron/electron-quick-start, did you mean to link another repo?

**Comment by littlecold233** [comment]
> @khoaakt there is no webview directory in https://github.com/electron/electron-quick-start, did you mean to link another repo?

Well, I'm not able to push to `electron-quick-start` repo. But you could reproduce the issue in your local repo

![image](https://user-images.githubusercontent.com/20556833/52101557-26e29e80-2617-11e9-85fc-f242ea60f504.png)

Just add this line code to the `index.html` in  `electron-quick-start`, the `src` can be any website with a `video` tag. Click the fullscreen button, you will see the problem

```
    <webview style="height: 1000px; width: 100%;" src="https://www.youtube.com/watch?v=OSUxrSe5GbI"></webview>

```

**Comment by 4eveRS** [comment]
Same problem.

**Comment by khoaakt** [comment]
> @khoaakt there is no webview directory in https://github.com/electron/electron-quick-start, did you mean to link another repo?

Yes, sorry for my mistake. Just edited the repo.
https://github.com/hokein/electron-sample-apps.git

Thanks.

**Comment by steverandy** [comment]
I can also reproduce the issue on youtube website. Using v4.0.4 and on macOS 10.14.2

**Comment by tassoevan** [comment]
Is there any known workaround?

**Comment by tassoevan** [comment]
Maybe #16428 refers to the same issue.

**Comment by littlecold233** [comment]
@codebytere Really need help. This problem is very serious

**Comment by 4eveRS** [comment]
This issue seems to be fixed at v5.0.X.
